__tests__/api.test.js

```diff
@@ -27,4 +27,34 @@ describe('GET', () => {
                 })
         });
     });
-});
\ No newline at end of file
+    describe.only('GET reviews', () => {
+        test('200: should return an array of the catagories', () => {
+            return request(app)
+                .get('/api/reviews/2')
+                .expect(200)
+                .then(({ body }) => {
+                    const review = body
```

Comment:
You should be sending the review back on the **key** of `review` which sits on the key of `body`
e.g. `{ body : { review : { .... } } }`